app/views/editor/_editor.html.erb

```diff
@@ -3,32 +3,10 @@
 <div class="form-group dropzone">
   <textarea aria-label="Wiki Text" name="body" tabindex="2" class="form-control" id="text-input" rows="20" cols="60"><% if @node && @node.latest && @node.latest.body %><%= @node.latest.body %><% else %><%= params[:body] %><% end %></textarea>
   <div id="imagebar">
-    <div style="display:none;" id="create_progress" class="progress progress-striped active pull-right">
-      <div id="create_progress-bar" class="progress-bar" style="width: 0%;"></div>
+    <div id="create_progress" style="display:none;width:100%;" class="progress progress-striped active">
+      <div class="create_progress-bar" role="progressbar" style="width: 0%;"></div>
     </div>
-    <p>
```

Comment:
Perhaps we have to make this more accurately match working upload image bars like in comments, to get it to work again?

Reply:
For example:

https://github.com/publiclab/plots2/blob/c229df48ffca69038f5b26f5cda2baa56ee94634/app/views/comments/_form.html.erb#L48-L76

Reply:
Actually, see for reference @NitinBhasneria's work here! https://github.com/publiclab/plots2/pull/8240/files Just so you can see how the system test walks us through the process with some working code.